bitcoind-regtest/rpc-explorer/Main.hs

```diff
@@ -27,7 +27,7 @@ import System.Process (readProcessWithExitCode)
 import Bitcoin.Core.Regtest (
     NodeHandle,
     nodeAuth,
-    nodePort,
```

Comment:
Maybe a dumb question, but this seems like a breaking change. Would you not need to bump the version on this package? Are you gonna do a follow up or something? Or are we just not worried about versions for now?

Reply:
Version 0.3 is unreleased, so this is still cool